src/app/components/login/login.jsx

```diff
@@ -18,6 +18,7 @@ import ExitToAppIcon from '@material-ui/icons/ExitToApp';
 import CreateIcon from '@material-ui/icons/Create';
 import SendIcon from '@material-ui/icons/Send';
 import NotInterestedIcon from '@material-ui/icons/NotInterested';
+import theme from '../../themes/theme';
```

Comment:
Why is this added?

Reply:
Isnt a theme provider & theme need to be provided to have breakpoints? this goes along with your comment above ^^

Reply:
This line is added to this file, which creates a variable in the file called `theme`, but that variable is never used. So whether it's needed for breakpoints or whatever, if the variable isn't being used in the file it's not needed in this file